### PR TITLE
Add trigger action for setting device orientation

### PR DESCRIPTION
**Pull request by ichan-mb**
Add `SetWindowOrientation` trigger action. With this trigger action, we can now specify different screen orientation of a `Page` in Navigation.

**Example:**

```
<Page>
     <Activated>
         <SetWindowOrientation To="LandscapeLeft" />
     </Activated>
<Page>
```

This PR contains:
- [x] Changelog
- [x] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Platform/iOS/SystemUI.uno [hunk 1]
```diff
@@ -320,5 +322,95 @@ namespace Fuse.Platform
...
+				}
+			}
+		}
+
+		[Foreign(Language.ObjC)]
+		static int GetCurrentScreenOrientation()
+		@{
+			if (@available(iOS 13, *))
+			{
+				 UIInterfaceOrientation mask = [[UIApplication sharedApplication].windows firstObject].windowScene.interfaceOrientation;
```
Comment (ichan-mb) [inline Source/Fuse.Platform/iOS/SystemUI.uno:361]: Travis failed on this line complaining about `windowScene`, the property of `windowScene` on UIWindow is only available on iOS 13

Comment (mortend) [reply Source/Fuse.Platform/iOS/SystemUI.uno:363]: https://github.com/fuse-open/fuselibs/blob/master/Source/Fuse.Common/Internal/iOSSystemFont.uno demonstrates how to test for iOS13.

Compile-time check:
```
#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0

#endif
```

Run-time check:
```
if (Fuse.iOSDevice.OperatingSystemVersion.Major >= 13) 
{
}
```

Comment (ichan-mb) [reply Source/Fuse.Platform/iOS/SystemUI.uno:363]: I thought using `@available(iOS 13, *)` is enough, well I try to replace with `#if` directive then

**Comment by AndrewEQ** [comment]
Hi guys, I found an issue: https://github.com/fuse-open/fuselibs/issues/1374

**Pull request merged**